assets/base.css

```diff
@@ -2001,7 +2001,7 @@ input[type='checkbox'] {
 }
 
 .quantity__rules {
-  margin-top: 1.2rem;
+  margin-top: 0.5rem;
```

Comment:
Hey Andrew. This is looks good to me. 

One thing, we use this class in the cart-drawer and that affects the visual part. I leave a short [video](https://screenshot.click/08-47-wr2k5-lvfw4.mp4).
<img width="747" alt="Screenshot 2023-08-08 at 4 37 24 PM" src="https://github.com/Shopify/dawn/assets/105315663/59c38195-5854-4871-9d12-8bb7748431c0">